src/info.ts

```diff
@@ -0,0 +1,31 @@
+import { fileExists } from "./utils.js";
+import execute from "./spawn.js";
+
+export interface InfoSettings {
+  /** The path for the encrypted pdf */
+  input: string;
+  /** The password for the encrypted pdf */
+  password?: string;
+}
+
+/**
+ * Gets the encryption info for a PDF
+ * @param payload The settings for info
+ * @returns The output of QPDF
+ */
+export const info = async (payload: InfoSettings): Promise<Buffer> => {
```

Comment:
As a start we could show whether the file is encrypted or not

```suggestion
export interface InfoOutput {
  isEncrypted: boolean;
}


/**
 * Gets the encryption info for a PDF
 * @param payload The settings for info
 * @returns The output of QPDF
 */
export const info = async (payload: InfoSettings): Promise<InfoOutput> => {
```